src/gestor_vehiculos.ts

```diff
@@ -0,0 +1,12 @@
+import { Vehiculo } from "./vehiculo";
+import { Pedido } from "./pedido";
+
+type VehiculoDetalle = [Vehiculo, Pedido[], boolean];
+
+class GestorVehiculos{
```

Comment:
El último commit que ha modificado esto sólo cambia el nombre y pone el nombre correcto. Si es eso lo que hace, no se debe usar ninguna explicación adicional. Y si es posible, menos frases que no aportan ninguna información como "Entidad GestorVehículos closes #13". Está claro que es una entidad, porque no es un objeto valor. A continuación se repite que se ha creado la entidad cuando lo que se ha hecho es cambiar el nombre; https://github.com/Morad02/F1Data/commit/558fa32a5dd490ebffaf9c534ae89c057f0ea4f1 pero se dice que sirve para asociar pedidos a vehículos cuando eso se ha hecho arriba, y en otro commit.

Reply:
Los commits que he hecho he modificado una linea o he eliminado para poder hacer los commits y así incluir las explicaciones. No sabía de otra forma de poder incluir las explicaciones del commit, ya que no puedo modificarlo una vez subido.

Reply:
Las explicaciones que tienes que incluir son de lo que estés haciendo. Si has cambiado algo por un error ortográfico, eso es lo que habría que explicar (aunque esto es obvio). Si vas a hacer modificaciones más en profundidad, o las que hagas en el futuro, ahí es donde tienes que aplicar las buenas prácticas.

Reply:
Tenéis que resolver *todos* los comentarios.

Reply:
Los nuevos commits que se han realizado a partir de este momento, se han hecho con las indicaciones correspondientes